mix.exs

```diff
@@ -13,11 +13,13 @@ defmodule SqlDust.Mixfile do
   end
 
   def application do
-    [applications: [:logger]]
+    [applications: [:logger, :benchfella]]
   end
 
   defp deps do
     [
+      {:exprof, "~> 0.2.0"},
```

Comment:
If we keep them in they should be optional so people aren't forced to add them to the project if possible.